install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,599 @@
+for APBs.
+
+.`registry` Section Configuration Options
+[options="header",cols="1,3,1"]
+|===
+
+|Field |Description |Required
+
+|`name`
+|The name of the registry. Used by the broker to identify APBs from this registry.
+|Y
+ 
+|`user`
+|The user name for authenticating to the registry.
+|N
+
+|`pass`
+|The password for authenticating to the registry. 
+|N
+
+|`org`
+|The namespace or organization that the image is contained in.
+|N
+
+|`type`
+|The type of registry. Available adapters are `mock`, `rhcc`, `openshift`, `dockerhub`, and `local_openshift`.
+|Y
+
+|`namespaces`
+|The list of namespaces to configure the `local_openshift` registry adapter with. By default, a user should use `openshift`.
+|N
+
+|`url`
+|The URL that is used to retrieve image information. Used extensively for RHCC while the `dockerhub` adapter uses hard-coded URLs.
+|N
+
+|`fail_on_error`
+|Should this registry fail, the bootstrap request if it fails. Will stop the execution of other registries loading.
+|N
+
+|`white_list`
+|The list of regular expressions used to define which image names should be allowed through. Must have a white list to allow APBs to be added to the catalog. The most permissive regular expression that you can use is `.*-apb$` if you would want to retrieve all APBs in a registry. See xref:oab-broker-config-apb-filtering[APB Filtering] for more details.          
+|N
+
+|`black_list`
+|The list of regular expressions used to define which images names should never be allowed through. See xref:oab-broker-config-apb-filtering[APB Filtering] for more details.
+|N
+
+|`images`
+|The list of images to be used with an OpenShift Container Registry.
+|N
+|===
+
+[[oab-broker-config-registry-production]]
+=== Production
+
+A _production_ broker configuration is designed to be pointed at a trusted
+container distribution registry, such as the Red Hat Container Catalog (RHCC).
+
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: http://rhcc.redhat.com/api
+    user: <user>
+    pass: <password>
+  - type: local_openshift
+    name: lo
+    namespaces:
+      - openshift
+----
+
+[[oab-broker-config-registry-development]]
+=== Development
+
+A _development_ broker configuration is primarily used by developers working on
+the broker. Set the registry name to `dev` and the `devbroker` field to `true`
+to enable developer settings.
+
+[source,yaml]
+----
+registry:
+  name: dev
+----
+
+[source,yaml]
+----
+broker:
+  devbroker: true
+----
+
+[[oab-broker-config-registry-mock]]
+=== Mock Registry
+
+Using a mock registry is useful for reading local APB specs. Instead of going
+out to a registry to search for image specs, this uses a list of local specs.
+Set the name of the registry to `mock` to use the mock registry.
+
+[source,yaml]
+----
+registry:
+  - name: mock
+    type: mock
+----
+
+[[oab-broker-config-registry-dockerhub]]
+=== Dockerhub Registry
+
+Using the `dockerhub` registry allows you to load APBs from a specific
```

Comment:
s/Using the `dockerhub`/The `dockerhub` 

For consistency and brevity?

Reply:
:+1: